app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -0,0 +1,61 @@
+package fr.free.nrw.commons.feedback;
+
+import android.content.Context;
+import fr.free.nrw.commons.feedback.Feedback;
+
+public class FeedbackContentCreator {
+    private StringBuilder stringBuilder;
+    private Feedback feedback;
+
+    public FeedbackContentCreator(Feedback feedback) {
+        this.feedback = feedback;
+        init();
+    }
+
+    /**
+     * Initializes the string buffer object to append content from feedback object
+     */
+    public void init() {
+        // Localization is not needed here, because this ends up on a page where developers read the feedback, so English is the most convenient.
+
+        stringBuilder = new StringBuilder();
+        stringBuilder.append("== ").append("Feedback for version").append(feedback.getVersion()).append(" ==");
+        stringBuilder.append("\n");
+        stringBuilder.append(feedback.getTitle());
+        stringBuilder.append("\n");
+        stringBuilder.append("\n");
+        if(feedback.getApiLevel()!=null) {
```

Comment:
The Java convention is to places spaces before and after `!=`, and also after `if`.
So here: `if (feedback.getApiLevel() != null) {`
Could you please also apply the same in other places of your code?

It may sound futile, but it really makes the code easier to read.
https://www.oracle.com/java/technologies/javase/codeconventions-whitespace.html